TwoNote/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Jul 21 08:59:00 EDT 2020
+#Thu Jul 23 14:36:48 EDT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```

Comment:
is this what fixed your MainActivity issue?

Reply:
yeah this and clicking File > Invalidate cache/restart